proto/interchain_security/ccv/provider/v1/query.proto

```diff
@@ -184,13 +183,14 @@ message Chain {
   // Corresponds to a list of provider consensus addresses of validators that CANNOT validate the consumer chain.
   repeated string denylist = 8;
   // The phase the consumer chain (Registered=0|Initialized=1|FailedToLaunch=2|Launched=3|Stopped=4)
-  ConsumerPhase phase = 9;
+  string phase = 9;
```

Comment:
Can you pls update the comment above (outdated enum values) and I assume we return
"Registered" and not "0" so probably removing completely the wrong enums would be good
```suggestion
  // The phase of the consumer chain ("Registered"|"Initialized"|"Launched"|"Stopped")
  string phase = 9;
```